dev_guide/output_deps.adoc

```diff
@@ -5,50 +5,64 @@
 :icons:
 :experimental:
 :toc: macro
-:toc-title: 
+:toc-title:
 
 toc::[]
 
 == Overview
-`build-chain` is an experimental command-line tool for OpenShift v3. This tool will start at the imagerepo:tag specified and determine what Builds will be triggered (due to ImageChangeTriggers on the BuildConfig) when this imagerepo:tag is updated. Further, it analyzes the output of those Builds to determine if they in turn update another imagerepo:tag. If they do, the tool continues to follow the dependency tree. It then outputs the total graph of what imagerepo:tags will be impacted by an update to the top level imagerepo:tag. The default output is set to JSON but the user can also choose between DOT and AST.
- 
- 
-== Examples
+*build-chain* is an experimental command-line tool for OpenShift v3, which
+allows you to identify which images would be affected by updating an image in a
+specified image stream.
 
+This tool starts at a specified image stream tag and determines which builds
+to trigger when that tag is updated (based on ImageChangeTriggers on the
+BuildConfig). It then analyzes the output of those builds to determine if they
+will in turn update another image stream tag. If they do, the tool continues
+to follow the dependency tree. It then outputs the total graph which specifies
+the image stream tags that would be impacted by an update to the top-level
+tag. The default output syntax is set to JSON, but DOT and AST are also
+available.
 
-Build dependency tree for the specified image repository with the specified tag in the current namespace and output it in JSON
-----
-$ openshift ex build-chain [image-repository]:[tag]
-----
+== Usage
 
-Build and output the dependency tree for the specified image repository with 'latest' tag in 'testing' namespace using DOT syntax
-----
-$ openshift ex build-chain [image-repository] -o dot -n testing
-----
+Build a dependency tree for the specified image stream with the specified
+tag in the current namespace and output it in JSON:
+****
+`$ openshift ex build-chain _<image-stream>_:_<tag>_`
```

Comment:
To render properly, this should be (like the one near the bottom):

`_<image-stream>:<tag>_`